sympy/printing/tests/test_llvmjit.py

```diff
@@ -105,6 +105,26 @@ def test_callback_cubature():
     assert isclose(out_array[0], res)
 
 
+def test_callback_cubature_v():
+    e = a + 1.2
+    f = g.llvm_callable([a], e, callback_type='cubature_v')
+    ndim = ctypes.c_int(1)
+    fdim = ctypes.c_int(1)
+    npts = ctypes.c_size_t(2)
+    array_type = ctypes.c_double * 2
+    inp = [2.2, 2.3]
+    array = array_type(inp[0], inp[1])
+    out_array = array_type(0.0, 0.0)
+    jit_ret = f(ndim, npts, array, None, fdim, out_array)
+
+    assert jit_ret == 0
+
+    res = [float(e.subs(a, x).evalf()) for x in inp]
+
+    assert isclose(out_array[0], res[0])
+    assert isclose(out_array[1], res[1])
+
+
 def test_callback_two():
     e = 3*a*b
     f = g.llvm_callable([a, b], e, callback_type='scipy.integrate.test')
```

Comment:
There is not a direct dependency for this test (that is, there is no need to import scipy).  The test here calls the callback function directly (via ctypes).   There is an implicit dependency in that the definition of the callback function (order and meaning of parameters) is the same here and in scipy.integrate.   A complete test would import scipy and call the integration routine with the callback (one of the doctests for this function does this).